src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -783,6 +785,9 @@ protected override TypeWithAnnotations InferTypeOfVarVariable(DiagnosticBag diag
                         var armBinder = (SwitchExpressionArmBinder)_nodeBinder;
                         armBinder.BindSwitchExpressionArm(arm, diagnostics);
                         break;
+                    case SyntaxKind.GotoCaseStatement:
+                        _nodeBinder.BindStatement((GotoStatementSyntax)_nodeToBind, diagnostics);
```

Comment:
>_nodeBinder.BindStatement((GotoStatementSyntax)_nodeToBind, diagnostics); [](start = 24, length = 73)

Was this breaking a test?

Reply:
Yes, the new one. It also broke bootstrap builds but only in some non-Windows configuration, but I was not able to easily diagnose why.